setup-database.sh

```
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+echo "🚀 Setting up PDF Export Database..."
+
+# Load environment variables from .env file if it exists
+if [ -f .env ]; then
+    echo "📄 Loading environment variables from .env file..."
+    export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)